xla/service/gpu/autotuning/gemm_fusion_autotuner_test.cc

```diff
@@ -306,13 +323,26 @@ class GemmFusionAutotunerTest : public StatelessAutotunerTest {
     return debug_options;
   }
 
+  stream_executor::GpuComputeCapability CudaAmpereOrRocm() {
```

Comment:
This function can be somewhat confusing. I suggest you simplify the logic and directly inline it in the single place where it is used:

```
if (IsRocm()) {
return GetRocmComputeCapability();
} else {
...
}
```
I've another comment suggesting you introduce `IsRocm()`, and it's probably also good to have `GetRocmComputeCapability()`, similarly to how we have a function for cuda.

Reply:
Once you introduce those two new function, please make a pass over the entire file and use them where appropriate.